om4labs/diags/generic_yz_annual_bias_1x1deg/generic_yz_annual_bias_1x1deg.py

```diff
@@ -0,0 +1,385 @@
+#!/usr/bin/env python
+
+import io
+import numpy as np
+import argparse
+import xarray as xr
+import warnings
+import pkg_resources as pkgr
+import intake
+
+from om4labs import m6plot
+from om4labs.helpers import get_run_name, try_variable_from_list
+from om4labs.om4plotting import plot_yzdiff, plot_yzcompare
+from om4labs.om4common import read_data, subset_data
+from om4labs.om4common import simple_average, copy_coordinates
+from om4labs.om4common import compute_area_regular_grid
+from om4labs.om4common import DefaultDictParser
+from om4labs.om4common import image_handler
+
+from cmip_basins import generate_basin_codes
+
+imgbufs = []
+
+
+def read(dictArgs):
+    """ read data from model and obs files, process data and return it """
+
+    if dictArgs["model"] is not None:
```

Comment:
config?

Reply:
This PR has been hanging around for a few weeks so I'd rather close the book on this first, then update all of the parsers at one time on a new branch.

Reply:
I don't see "model" in the parser so I don't think this will work and is this option relevant to the 1x1deg plots?

Reply:
It's there ... [Line 174](https://github.com/raphaeldussin/om4labs/blob/fb30f2df7a87284cacaec7dc9be81b7cc16720ba/om4labs/diags/generic_yz_annual_bias_1x1deg/generic_yz_annual_bias_1x1deg.py#L174)

Reply:
Oh forgot to answer the second part.  Yes it's relevant because cell area is still model dependent even on the 1x1º grid.

Reply:
I'm not sure to follow